packages/config/src/options.js

```diff
@@ -395,5 +395,10 @@ export function getNuxtConfig (_options) {
     options.server.timing = { total: true, ...timing }
   }
 
+  if (!Array.isArray(options.serverMiddleware)) {
```

Comment:
Maybe `isPureObject` from utils would be better suited? It only doesnt check for _null_, if you check for that as well then Object.entries will be guaranteed to only be called on a proper object.

Reply:
Nice, forgot about `isPureObject()`, thanks.